src/OpenTok/OpenTok.php

```diff
@@ -255,6 +255,103 @@ public function createSession($options = array())
         ));
     }
 
+    /**
+     * This method starts an Experience Composer Render session.
+     *
+     * @param $sessionId (string) The session ID.
+     * @param $token (string) A valid OpenTok token with a Publisher role and (optionally) connection data to be associated with the output stream.
+     * @param $url (string) A publicly reachable URL controlled by the customer and capable of generating the content to be rendered without user intervention.
+     * @param $maxDuration (int) (optional) The maximum time allowed for the Render, in seconds. After this time, the Render will be stopped automatically, if it is still running.
+     * @param $resolution (string) (optional) Resolution of the display area for the composition.
+     * @param $statusCallbackUrl (string) (optional) URL of the customer service where the callbacks will be received.
```

Comment:
We are not documenting this REST API option. Developers should use account portal to register callbacks. See https://tokbox.com/developer/guides/experience-composer/#configuring-callbacks.